### PR TITLE
Worked on Question Box Feature 

### PR DESCRIPTION
**Pull request by adhithiganesan**
I created a place where the user can enter a question which is then displayed as h1 and they can also update their question. It's not perfect but I got the basic things working. I think it still needs:
- design/css code to position the buttons/text areas better
- I think a nice feature to include would be when the user updates the question, the previous question they had asked is in the textarea, ready for them to edit and update it. I can work on this feature or anyone else can too if they're interested.

### CODE REVIEW

File: Fortune-Teller/css/style.css [hunk 1]
```diff
@@ -1,7 +1,40 @@
+h1{
```
Comment (mojeanmac) [inline Fortune-Teller/css/style.css:1]: This is conflicting with the other menu h1 tags
for example, the landing page has beige text that is no longer readable due to the text background
image

**Review by mojeanmac — COMMENTED** [review]
It looks good but the css is interfering with other pages (specifically h1 tag)

The next step would be to make sure the css fits with the theming and colors of the website

File: Fortune-Teller/css/style.css (hunk 1 above)
Comment (adhithiganesan) [reply Fortune-Teller/css/style.css:1]: Yikes sorry I didn't realize that, thank you for catching it! I'll go through and change the CSS soon

**Comment by adhithiganesan** [comment]
i updated it to not use the h1tag

**Review by c5du — APPROVED** [review]
Nice job!

**Pull request merged**